### ISSUE TITLE
Something's broken...

### ISSUE THREAD

**Issue by robsheldon** [programical/gitland#98]
...or I got nerfed? :-)

It looks like a lot of the bots that were playing well aren't working correctly now. In my bot's case, it keeps trying to issue moves right:

```
$ git log -p act | grep "^\+[a-z]" | head -n 20
+right
+idle
+right
+up
+right
+up
+right
+up
+right
+down
+right
+up
+right
+up
+right
+up
+right
+up
+right
+up
```

...but the server keeps moving the bot left:
```
$ git log -p players/robsheldon/x | grep "^[-+][0-9]" | head -n 20
-1
+0
-2
+1
-3
+2
-2
+3
-3
+2
-2
+3
-1
+2
-0
+1
-1
+0
-0
+1
```

I don't see a spot in the server code that should be causing this or a caching issue on Github's end.

**Comment by programical** [comment]
It's not just you 0_0

**Comment by wmww** [comment]
I think my parallel requests PR is triggering GitHub's DDoS protection. When I run it a bunch of times It get this in my response HTML:
![a](https://user-images.githubusercontent.com/9331264/80871709-cc9e1680-8c7b-11ea-80bd-c8d5b7bdc7be.png)

**Comment by programical** [comment]
We could (maybe) fix this by merging #94 but that still has unresolved conflicts.

**Comment by robsheldon** [comment]
It might be better to take the approach in #94 anyway if you think the game might grow much more.
